README.md

```diff
@@ -1,30 +1,78 @@
-# Juju Charm/Operator for Prometheus on Kubernetes
+# Prometheus Operator
 
 ## Description
 
-[Prometheus](https://prometheus.io) is an open source monitoring
-system and alterting toolkit. This repository contains a
-[Charm](https://discourse.juju.is/t/charm-writing/1260) for deploying
-Prometheus in Kubernetes clusters using [Juju](https://jaas.ai/).
+The Prometheus Operator provides a cluster monitoring solution using
+[Prometheus](https://prometheus.io), which is an open source
+monitoring system and alerting toolkit.
 
+This repository contains a [Juju](https://jaas.ai/) Charm for
+deploying the monitoring component of Prometheus in a Kubernetes
+clusters. The alerting component of prometheus is offered through a
+separate Charm.
+
+## Setup
+
+A typical setup using [snaps](https://snapcraft.io/), for deployments
+to a [microk8s](https://microk8s.io/) cluster can be done using the
+following commands
+
+    sudo snap install microk8s --classic
+    microk8s.enable dns storage registry dashboard
```

Comment:
Why registry?

Reply:
Not essential but standard practice since that may be used for container images.